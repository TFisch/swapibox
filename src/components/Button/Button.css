button.Planets,button.Vehicles,button.People,button.FAVORITES {
  background-color: #1F2833;
  border: 3px solid #D7AA39;
  color: #66FCF1;
  font-family: "Fjalla One", sans-serif;
  width: 10em; }
