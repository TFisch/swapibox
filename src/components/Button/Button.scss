button.card{
  height: 20px;
  width: 20px;
}