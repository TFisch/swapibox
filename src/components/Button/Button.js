import React from 'react'
import './Button.css'

const Button = ({ onClick, name, favorites }) => {
  return (
    <button 
      className={favorites ? 'favoriteBtn' : {name}} 
      onClick={onClick}
      value={name}
    >
    {name}
    </button>
  )
}
export default Button