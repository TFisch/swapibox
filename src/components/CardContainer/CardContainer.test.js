import React from 'react';
import { shallow } from enzyme;
import Button from '../CardContainer/CardContainer';

