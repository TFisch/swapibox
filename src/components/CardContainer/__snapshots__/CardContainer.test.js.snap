// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer Should match the snapshot when displayData has length 1`] = `"<div class=\\"card-container\\"><div class=\\"card\\"><span class=\\"card-title\\"><h1 class=\\"name\\">Luke Skywalker</h1><button class=\\"cardbtn\\"><img class=\\"imgcardbtn\\" src=\\"mark.svg\\"/></button></span><p class=\\"stat\\"><p class=\\"individual-stat\\">Homeworld: [object Object]</p><p class=\\"individual-stat\\">Population: [object Object]</p><p class=\\"individual-stat\\">Species: [object Object]</p></p></div></div>"`;

exports[`CardContainer Should match the snapshot when favorites is displaying and has no length 1`] = `"<div class=\\"card-container\\"><div class=\\"card\\"><span class=\\"card-title\\"><h1 class=\\"name\\">Luke Skywalker</h1><button class=\\"cardbtn\\"><img class=\\"imgcardbtn\\" src=\\"mark.svg\\"/></button></span><p class=\\"stat\\"><p class=\\"individual-stat\\">Homeworld: [object Object]</p><p class=\\"individual-stat\\">Population: [object Object]</p><p class=\\"individual-stat\\">Species: [object Object]</p></p></div></div>"`;

exports[`CardContainer Should match the snapshot when only crawlingText has length 1`] = `
"<div class=\\"card-container\\"><div class=\\"fade\\"></div><section class=\\"star-wars\\"><div class=\\"crawl\\"><div class=\\"title\\"><p>Episode 4</p><h1>A New Hope</h1></div><p>It is a period of civil war.
Rebel spaceships, striking
from a hidden base, have won
their first victory against
the evil Galactic Empire.

During the battle, Rebel
spies managed to steal secret
plans to the Empire&#x27;s
ultimate weapon, the DEATH
STAR, an armored space
station with enough power
to destroy an entire planet.

Pursued by the Empire&#x27;s
sinister agents, Princess
Leia races home aboard her
starship, custodian of the
stolen plans that can save her
people and restore
freedom to the galaxy....</p><p>Release Date: </p></div></section></div>"
`;
